ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -209,6 +210,28 @@
   [context]
   (bulk-update/cleanup-old-bulk-update-status context))
 
+(defn email-granule-url-list
```

Comment:
use `defn-` for private functions.

Reply:
can you test functions that are defined with defn-